pkg/cmd/kafka/consumergroup/describe/describe.go

```diff
@@ -144,13 +141,10 @@ func runCmd(opts *options) error {
 	}
 
 	stdout := opts.IO.Out
+
 	switch opts.outputFormat {
-	case dump.JSONFormat:
-		data, _ := json.Marshal(consumerGroupData)
-		_ = dump.JSON(stdout, data)
-	case dump.YAMLFormat, dump.YMLFormat:
-		data, _ := yaml.Marshal(consumerGroupData)
-		_ = dump.YAML(stdout, data)
+	case dump.JSONFormat, dump.YAMLFormat, dump.YMLFormat:
```

Comment:
Switch with 1 case?

Reply:
We can still use switch, but make it simpler. See https://github.com/redhat-developer/app-services-cli/pull/1066#discussion_r710011053

Reply:
Done